auditlog/models.py

```diff
@@ -208,6 +212,18 @@ def _get_pk_value(self, instance):
             pk = self._get_pk_value(pk)
         return pk
 
+    def _get_serialized_data_or_none(self, instance):
+        from .registry import auditlog
+
+        registry_opts = auditlog.get_model_fields(instance.__class__)
+        if not registry_opts["serialize_data"]:
+            return None
+
+        kwargs = registry_opts.get("serialize_kwargs", {})
+        kwargs.setdefault("fields", [x.name for x in instance._meta.fields])
+        data = json.loads(serializers.serialize("json", (instance,), **kwargs))
+        return next(iter(data), None)
```

Comment:
What is the case in which `data` can be empty?

Reply:
Probably never. I've changed the approach here.